lib/ansible/plugins/doc_fragments/constructed.py

```diff
@@ -22,7 +22,10 @@ class ModuleDocFragment(object):
     type: dict
     default: {}
   groups:
-    description: Add hosts to group based on Jinja2 conditionals.
+    description: >
+      Add hosts to group based on Jinja2 conditionals.
+      A list of filters can be found here
+      U(https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html).
```

Comment:
Instead of using an external link (which might change and point to `latest` version) please use an internal link https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role

Reply:
Documentation of the correct formats for internal links: https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation.

Reply:
I fixed as requested, please let me know if that's what you expect.
Thanks !